share/spack/gitlab/cloud_pipelines/stacks/aws-e4s-aarch64/spack.yaml

```diff
@@ -0,0 +1,451 @@
+   #- dealii +cuda cuda_arch=70 # gmsh
+   #- flecsi +cuda cuda_arch=70
+   #- paraview +cuda cuda_arch=70
+
+  - rocm_specs:
+    - kokkos +rocm amdgpu_target=gfx906
+    #- amrex +rocm amdgpu_target=gfx906
+    #- chai +rocm ~benchmarks amdgpu_target=gfx906
+    #- ginkgo +rocm amdgpu_target=gfx906 # needs hip<4.1
+    #- raja +rocm ~openmp amdgpu_target=gfx906 # blt 0.3.6 issue with rocm
+    #- slate +rocm amdgpu_target=gfx906
+    #- strumpack +rocm ~slate amdgpu_target=gfx906
+    #- sundials +rocm amdgpu_target=gfx906
+    #- tasmanian +rocm amdgpu_target=gfx906
+    #- umpire+rocm amdgpu_target=gfx906 # blt 0.3.6 issue with rocm
+
+  - default_specs:
+    - adios
+    - adios2
+    - aml
+    - amrex
+    - arborx
+    - archer
+    - argobots
+    - ascent
+    - axom
+    - bolt
+    - cabana
+    - caliper
+    - chai ~benchmarks ~tests
+    - conduit
+    - darshan-runtime
+    - darshan-util
+    - datatransferkit
+    - dyninst
+    - faodel
+    - flecsi@1.4.2 +external_cinch
+    - flit
+    - flux-core
+    - fortrilinos
+    - gasnet
+    - ginkgo
+    - globalarrays
+    - gmp
+    - gotcha
+    - gptune
+    - h5bench
+    - hdf5
+    - heffte +fftw
+    - hpctoolkit
+# x86 specific
+    #- hpx
+    - hypre
+    - kokkos +openmp
+    - kokkos-kernels +openmp
+    - lammps
+    - legion
+    - libnrm
+    - libquo
+    - libunwind
+# bad build on aarch64, graviton2 (likely amdgpu?)
+    #- llvm targets=amdgpu,nvptx +clang +compiler-rt +libcxx +lld +lldb +llvm_dylib +flang ~cuda
+    - loki
+    - mercury
+    - metall
+    - mfem
+    - mpark-variant
+    - mpifileutils ~xattr
+    - nccmp
+    - nco
+    - ninja
+    - nrm
+    - omega-h
+    - openmpi
+    - openpmd-api
+    - papi
+    - papyrus@1.0.1
+    - parallel-netcdf
+    - parsec ~cuda
+    - pdt
+    - petsc
+# Depends on emmintrin.h (x86)
+    #- phist
+    - plasma
+    - precice
+    - pumi
+    - py-jupyterhub
+    - py-libensemble
+    - py-petsc4py
+    - py-warpx ^warpx dims=2
+    - py-warpx ^warpx dims=3
+    - py-warpx ^warpx dims=rz
+    - qthreads scheduler=distrib
+    - raja
+    - rempi
+    - scr
+    - slate ~cuda
+    - slepc
+    - stc
+    - strumpack ~slate
+    - sundials
+    - superlu
+    - superlu-dist
+    - swig
+    - swig@4.0.2-fortran
+    - sz
+    - tasmanian
+    - tau +mpi +python
+    - trilinos +amesos +amesos2 +anasazi +aztec +belos +boost +epetra +epetraext +ifpack +ifpack2 +intrepid +intrepid2 +isorropia +kokkos +ml +minitensor +muelu +nox +piro +phalanx +rol +rythmos +sacado +stk +shards +shylu +stokhos +stratimikos +teko +tempus +tpetra +trilinoscouplings +zoltan +zoltan2 +superlu-dist gotype=long_long
+    - turbine
+    - umap
+    - umpire
+    - unifyfs@0.9.1
+    - upcxx
+# x86 specific ASM
+    #- variorum
+    - veloc
+    - wannier90
+    - zfp
+    #- dealii
+    #- geopm
+    #- qt
+    #- qwt
+    #- umpire # unsatisfiable concretization conflict w/ blt
+
+  - compiler:
+    - '%gcc'
```

Comment:
Why not specify the compiler version here?  Maybe it's no big deal, just wondering.

Reply:
updated